mix.exs

```diff
@@ -177,7 +177,7 @@ defmodule Logflare.Mixfile do
       {:floki, "~> 0.29.0", only: [:test]},
 
       # Rust NIFs
-      {:rustler, "~> 0.34.0"},
+      {:rustler, "~> 0.36.2", override: true},
```

Comment:
why is the override necessary?

Reply:
explorer is hardcoded upstream in main to rely on 34

Reply:
And I should note that the changes are safe between 0.34 and 0.36.2 in context of explorer.

https://hexdocs.pm/rustler/changelog.html